client/landing/subscriptions/comment-list/comment-row.tsx

```diff
@@ -0,0 +1,51 @@
+import { Gridicon } from '@automattic/components';
+import moment from 'moment';
+import { useMemo } from 'react';
+import type { PostSubscription } from '@automattic/data-stores/src/reader/types';
+
+const CommentRow = ( {
+	title,
+	excerpt,
+	site_title,
+	site_icon,
+	site_url,
+	date_subscribed,
+}: PostSubscription ) => {
+	const since = useMemo(
+		() => moment( date_subscribed ).format( 'LL' ),
+		[ date_subscribed, moment ]
+	);
+	const hostname = useMemo( () => new URL( site_url ).hostname, [ site_url ] );
+	const siteIcon = useMemo( () => {
+		if ( site_icon ) {
+			return <img className="icon" src={ site_icon } alt={ site_title } />;
+		}
+		return <Gridicon className="icon" icon="globe" size={ 48 } />;
+	}, [ site_icon, site_title ] );
+
+	return (
+		<li className="row" role="row">
+			<span className="post" role="cell">
+				<div className="title">{ title }</div>
+				<div className="excerpt">{ excerpt }</div>
+			</span>
+			<a href={ site_url } rel="noreferrer noopener" className="title-box" target="_blank">
+				<span className="title-box" role="cell">
+					{ siteIcon }
+					<span className="title-column">
+						<span className="name">{ site_title }</span>
+						<span className="url">{ hostname }</span>
+					</span>
+				</span>
+			</a>
```

Comment:
@phcp suggested that we could extract out the section with Site icon, title and URL into its own component. I like the idea and we could do it in a separate task as well.